frontend/src/components/user/RequireLogin.tsx

```diff
@@ -0,0 +1,26 @@
+"use client";
+
+import { useTranslations } from "next-intl";
+import { redirect, useSearchParams } from "next/navigation";
+import { Alert, GridContainer } from "@trussworks/react-uswds";
+
+export function RequireLogin() {
+  const t = useTranslations("Errors");
+  const searchParams = useSearchParams();
+
+  if (searchParams.get("auth_logout") != null) {
```

Comment:
can we make this more exact? I wouldn't want to redirect accidentally in edge cases here
```suggestion
  if (searchParams.get("redirectOnLogout") === "true") {
```